### DISCUSSION TITLE
What’s next?

### DISCUSSION THREAD

**Discussion by upintheairsheep** [TrungNguyen1909/qemu-t8030#79] [Q&A] [answered]
After finishing the iPhone 11 emulator, what will you emulate next? I already think you’ll emulate the other A13 devices, the iPhone SE 2 and the iPad 9th generation next. And probably even the Studio Display because it uses A13 and why not emulate it? What Apple chipset will you emulate next?

**Comment by TrungNguyen1909** [comment, marked as answer]
> After finishing the iPhone 11 emulator
This one is not going to happen very soon.

But anyway, to entertain your speculation, I would probably try to work on newer iPhone generations.

There is not much value in emulating similar devices (especially the Studio Display, which has no practical value at all).

**Reply by upintheairsheep** [reply]
I’m a little bummed you did not decide on working on older ones. 32 bit apps are something I’m excited about along with iOS 6.

**Reply by iarchiveml** [reply]
imagine you're entitled

**Comment by upintheairsheep** [comment]
I wished you would have done the opposite, by emulating older chips,
especially ones that support 32 bit.

**Reply by iarchiveml** [reply]
r/entitledkids

**Reply by TrungNguyen1909** [reply]
@iarchiveml yo, quit it. No need to start a flame war on a thread that has gone inactive a month ago.

**Reply by upintheairsheep** [reply]
I knew already there is https://systems.cs.columbia.edu/projects/cycada/ , which solves my problem, if only it was realeased. Next time I go on a trip to NYC, I'm going to talk to them in person, kindly requesting the release of the software, and solving any hurdles.
